flink-python/pyflink/table/tests/test_pandas_udf.py

```diff
@@ -165,6 +165,12 @@ def nested_array_func(nested_array_param):
                 'nested_array_param of wrong type %s !' % type(nested_array_param[0])
             return pd.Series(nested_array_param[0])
 
+        def row_func(row_param):
+            assert isinstance(row_param, pd.Series)
+            assert isinstance(row_param[0], dict), \
```

Comment:
I think this will be handled internally by Arrow, e.g. it could identify the order of the columns according to the field names and so I think there is no order problems. What do you think?